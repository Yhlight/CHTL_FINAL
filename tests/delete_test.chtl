html {
    head {
        title { text { "Origin Test" } }
    }
    body {
        h1 { text { "Content from CHTL" } }

        [Origin] @Html {
            <div class="raw-html">
                <p>This is <strong>raw, unprocessed</strong> HTML.</p>
                <!-- This is a raw HTML comment -->
            </div>
        }

        p { text { "More content from CHTL" } }
    }
}
