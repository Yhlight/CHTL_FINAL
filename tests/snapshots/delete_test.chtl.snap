<html>
  <head>
    <title>
      Origin Test
    </title>
  </head>
  <body>
    <h1>
      Content from CHTL
    </h1>
<div class="raw-html">
                <p>This is <strong>raw, unprocessed</strong> HTML.</p>
                <!-- This is a raw HTML comment -->
            </div>
            <p>
      More content from CHTL
    </p>
  </body>
</html>
